### ISSUE TITLE
Reboot sparky and sparky web from UI

### ISSUE THREAD

**Issue by thibaultduponchelle** [melezhik/sparky#19]
Ability to ask for a soft restart from UI.

This feature depends on #17

**Comment by melezhik** [comment]
yeah, I find it's rear we user might need it, but maybe you can think of more real life examples?

**Comment by melezhik** [comment]
probably useful, when someone change sparky config and want that sparky will apply those changes ...

**Comment by thibaultduponchelle** [comment]
The other use case I had in mind is about potential stuck web app but it probably requires more than a "soft" restart
